src/Types/Never_.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Reflection\Types;
+
+use phpDocumentor\Reflection\Type;
+
+/**
+ * Value Object representing the pseudo-type 'void'.
```

Comment:
Please update this description

Reply:
Also, never is not a pseudo-type, right? It will be an actual type?

Reply:
the docs of php are not updated yet with this information... I'm not sure if it will be an actual type.. since you cannot used it like other scalar types. it is only allowed for return types.